recipes/google-cloud-aiplatform/meta.yaml

```
@@ -0,0 +1,67 @@
+{% set name = "google-cloud-aiplatform" %}
+{% set version = "1.10.0" %}
+{% set sha256 = "b6901122638f13f7c4ec4218a4d41855766f056d169bb29377492175f384c5ed" %}
+{% set dev_url = "https://github.com/googleapis/python-aiplatform" %}
+{% set doc_url = "https://cloud.google.com/python/docs/reference/aiplatform/latest" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - google-api-core-grpc >=1.26.0,<3.0.0dev
+    - proto-plus >=1.10.1
+    - packaging >=14.3
+    - google-cloud-storage >=1.32.0,<2.0.0dev
+    - google-cloud-bigquery >=1.15.0,<3.0.0dev
```

The "extras" in https://github.com/googleapis/python-aiplatform/blob/v1.10.0/setup.py are a bit of a mess, but I think we can consolidate a few of those into a unified `run_constrained` section.

`run_constrained` is the closest we can get to "extras" https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#run-constrained It says "these are optional, but if you do have them installed, here are the compatible versions"

```suggestion
    - google-cloud-bigquery >=1.15.0,<3.0.0dev
    
  run_constrained:
    - tensorflow >=2.3.0,<=2.7.0
    - pandas >=1.0.0
    - pyarrow >=6.0.1
```

---

Thanks Tim!